jupyter/radon/stan/radon_np_uranium.stan

```diff
@@ -18,5 +18,6 @@ model {
   sigma ~ normal(0, 10);
 }
 generated quantities {
-  array[N] real y_rep = normal_rng(alpha[county] + beta[1] * x + beta[2] * u, sigma);
+  array[N] real y_rep = normal_rng(alpha[county] + beta[1] * x + beta[2] * u,
+                                   sigma);
```

Comment:
mostly curious, what caused this change?

Reply:
Line length. This file wasn't previously in the repository last time we ran the formatter

Reply:
Oh, that would do it :)